Makefile

```diff
@@ -317,6 +317,33 @@ clean-volumes:
 compose:
 	$(foreach MODULE, $(CANDIG_MODULES), $(MAKE) compose-$(MODULE);)
 
+#-- Temp --
+compose-authz:
+	docker-compose -f $(DIR)/lib/keycloak/docker-compose.yml up -d 2>&1
+	docker-compose -f $(DIR)/lib/vault/docker-compose.yml up -d 2>&1
+	docker-compose -f $(DIR)/lib/tyk/docker-compose.yml up -d 2>&1
+compose-authz-down:
+	docker-compose -f $(DIR)/lib/keycloak/docker-compose.yml down
+	docker-compose -f $(DIR)/lib/vault/docker-compose.yml down
+	docker-compose -f $(DIR)/lib/tyk/docker-compose.yml down
+compose-authz-clean: compose-authz-down \
+	# needs sudo to run;
```

Comment:
will need to review scripts, but sudo access will not be available in most deployment scenarios

Reply:
I could not find another way to remove the necessary directories. Any suggestions?